.appveyor.yml

```diff
@@ -2,6 +2,8 @@ environment:
   matrix:
   - julia_version: 1.0
   - julia_version: 1.1
```

Comment:
Do we want to bother testing on 1.1 and 1.2?

Reply:
Also, don't we want to remove this file in favour of using travis for everything? I'm fine to change that in a separate PR.